samples/nrf5340/netboot/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+cmake_minimum_required(VERSION 3.8.2)
```

Comment:
Bump this to 3.13 or whatever is used elsewhere

Reply:
bumped to 3.13.1